mne/io/reference.py

```diff
@@ -123,6 +125,80 @@ def _apply_reference(inst, ref_from, ref_to=None, copy=True):
     return inst, ref_data
 
 
+def add_eeg_reference(inst, ref_channels, copy=True):
```

Comment:
A user would not intuitively understand the difference between `set_eeg_reference` and `add_eeg_reference` by looking at the function

how about `add_reference_channel`?